server/controllers/notification.controllers.js

```diff
@@ -0,0 +1,73 @@
+import models from '../models';
+import serverError from '../helpers/server-error';
+
+const { Notification } = models;
+
+/**
+ * @description Get a specific notification and update the is_seen column.
+ * @param {*} req
+ * @param {*} res
+ * @returns {object} user provides their userId as params
+ */
+const updateIsSeenColumn = async (req, res) => {
+  try {
+    const { articleId } = req.params;
+    const notification = await Notification.findOne({
+      where: {
+        user_id: req.user.userObj.id,
+        article_id: articleId,
+      },
+    });
+
+    if (!notification) {
+      return res.status(200).json({
+        message: 'No notification found for this article',
```

Comment:
`notifications` @AnayoOleru

Reply:
> `notifications` @AnayoOleru
For clarity, is it for the database 'notification', or the one on the response 'message'? Why I didn't pluralize it, is because the function is supposed to only return one notification

Reply:
I see